src/MoBi.Presentation/Presenter/EditEventBuilderPresenter.cs

```diff
@@ -19,20 +19,22 @@
 using OSPSuite.Core.Services;
```

Comment:
This whole code is a mess. REtrieving object from context etc... 
I don't want to touch it too much because we'll probably revisit event stuff eventually...but wow